viktor_dev_tools/tools/subdomain.py

```diff
@@ -35,6 +35,22 @@ class EntityDict(TypedDict, total=False):
     parent_entity_type: int
 
 
+class UserDict(TypedDict, total=False):
```

Comment:
please rename this class, UserDict is already in use

Reply:
please make sure that running dev-tools locally from cli.py still works, it's throwing the following error:
ImportError: cannot import name 'UserDict' from 'viktor_dev_tools.tools.subdomain' (C:\WORK\PYTHON\viktor-apps\development-tools\venv\Lib\site-packages\viktor_dev_tools\tools\subdomain.py)